### PR TITLE
Rwd mvp

### LINKED ISSUE #1794: Pioneer Responsive Design
# Context 

The product was designed with no consideration of responsiveness, and while the developers have done some work to fill in some gaps, but largely the product does not work for example on mobile devices. 

Screenshot 2021-11-19 at 10 47 44

We thought the the primary device for users would be the desktop, for example because the mobile browsers generally do not support extensions. However, it's also clear that some users will end up viewing Pioneer from a mobile device, for example because they are just linked to some screen which has some information, and currently the product would entirely break in many such cases.

⚠️ Focus: > small desktops/ not full screen/ find all places that can be tuned for removal of overflowing content
🆕 : Mobile user agents. 

# Scope
Suggest to split the scope into 2 parts

## Part 1 - MVP -> Focus 🎯 

Does not include connecting wallet on mobile phone and signing transactions. Only viewing of the pages is concerned here. 

Specifically let's go through all of the modules and suggest the Responsive Updates for the existing features:

- My Profile
- Working Groups
- Proposals
- Council
- Election
- Forum
- Members
- Settings

Break Points in scope: 
xxs - 320px
xs - 425px
sm - 768px
md - 1024px
lg - 1440px

## Part 2 - Out of scope for now ⛔ (let's do it later)

Out of scope breakpoints:
xl - 1920px
xxl - 2560px

Update the design of the transactional flows
 **Forum**
 - Post to forum
 **Proposals**
 - Create proposal
 - Vote on proposal
 **Council/ Elections**
 - Apply to council
 - Vote for council candidate
 - Reveal vote
 **Working groups**
 - Create opening
 - Apply to opening
  
  # Design Support
  
Latest Figma assets for Pioneer

[Wireframes >](https://www.figma.com/files/project/25387558/Wireframes?fuid=1039560990590227876)
[Designs >](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-design?node-id=0-1&t=ig1SfT4xyELPGJl7-0) 
[Storybook >](https://pioneer-2-storybook-git-carthage-joystream.vercel.app/?path=/story/common-tokenvalue--default&args=value:000+000000000)

⚠️ Link to Example Figma (for different app, but use it as structure example) [LINK> ](https://www.figma.com/file/zQYsZXJo6jmbqcj1kRRmDG/Marketplace-page?node-id=0-1&t=X15wHkuMSu9ifjPV-0)

You may find: 
- readme (changelog)
- Design screens for default resolution
- RWD separately for each of the breakdpoint / size
- User stories (here will be applicable for only a handful of views, like from list to details, or navigating forum)
- Local components (some may need to have a responsive variation fleshed out)

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201958687417145/1201958993176630) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by eshark9312**
resolves #1794

### CODE REVIEW

**Comment by eshark9312** [comment]
I created mobile navigation bar and sidebar slider.
And fixed Council Page to be responsive.

**Review by chrlschwb — APPROVED** [review]

**Review by thesan — CHANGES_REQUESTED** [review]
Hello @eshark9312. For about a week I was waiting for a day that I could free to review this huge PR but I don't think it's going to happen. Instead this PR really needs to be split. But it shouldn't require too much efforts:

In short I'm suggesting that you add a new branch on an earlier commit, then push it to make a new (and much smaller) PR from it.

For example I noticed that the first 10 commits are about the side bar and the council section. It might make sense to group these first 10 commits together has one PR. To do so you can run:
```shell
git branch rwd/sidebar-and-council 4e6e081c114600c4e22f1bb9a3b1d909cba68b82
```
Then you can push this branch and create a new PR with it. I'll create a new `rwd` branch to merge this new PR too after I approve it. After that you can create a new branch, again with some commits that it would make sense to group together. This time the `rwd` branch will have to be pulled into this new branch. Then the branch should be pushed to create another PR. And so on...

Of course the first 10 commits is just an example, it's up to you to choose which commits to group together but please keep the number of changes on the PR small so it can go relatively smoothly through several review cycle if needed.

cc @chrlschwb

**Pull request closed without merging**
